index.js

```diff
@@ -50,6 +51,46 @@ client.on('message', message => {
         .send(MESSAGE_PREFIX + "Send '!verify' to access other channels")
         .catch(reason => console.log(reason))
     }
+    else if (text.startsWith(CONFIG.SELF_JOIN_COMMAND) || is_self_join_alias) {
```

Comment:
can we do regex matching here? If you're new to regex I'm happy to do the change :)

Reply:
Latest commit now parses this with regex